kalite/updates/management/commands/videodownload.py

```diff
@@ -11,12 +11,11 @@
 from django.utils.translation import ugettext as _
```

Comment:
Edits in here are mostly to replace the use of the VideoFile model to track download progress.